share/workspaces/Advanced.xml

```diff
@@ -1363,37 +1363,37 @@
         <mag>1</mag>
         <grid>1</grid>
         <moreElements>0</moreElements>
-        <Cell name="Arpeggio">
+        <Cell name="Arpeggio Normal">
           <Arpeggio>
             <subtype>0</subtype>
             </Arpeggio>
           </Cell>
-        <Cell name="Arpeggio">
+        <Cell name="Arpeggio Up">
```

Comment:
Better to call this "Up Arpeggio", and similar for the rest. That way blind users with screen readers don't have to listen to the whole name before they can decide whether it is the one they need, or whether to move to the next one.